core/run-service.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+service_command=$1
+memory_limit_mb=$2
```

Comment:
I think you're limiting services to 2MB 👀

Reply:
NO U!

Reply:
I added a debug line to be sure of what is going on